django/contrib/gis/geoip2/base.py

```diff
@@ -9,6 +9,7 @@
 
 from .resources import City, Country
 
+
```

Comment:
why this

Reply:
By defining the dictionary here the settings are loaded at import time. This means the `override_settings` decorator does not work, so we need to move this definition into the constructor.